python/ray/tune/stopper.py

```diff
@@ -67,7 +67,7 @@ def is_valid_function(cls, fn):
 
 
 class EarlyStopping(Stopper):
-    def __init__(self, metric, std=0.001, top=10, mode="min"):
+    def __init__(self, metric, std=0.001, top=10, mode="min", patience=0):
         """Create the EarlyStopping object.
```

Comment:
BTW, may you can add a bit about what this object does? 

For example, "Stops the entire experiment when the metric has plateaued."

Reply:
Sure!